.github/workflows/update-deps.yml

```diff
@@ -0,0 +1,41 @@
+name: Update Dependencies
+
+on:
+  # Run every day.
+  schedule:
+    - cron: '0 3 * * *'
+  # And on on every PR merge so we get the updated dependencies ASAP, and to make sure the changelog doesn't conflict.gi
+  push:
+    branches:
+      - main
+
+jobs:
+  cli:
+    uses: getsentry/github-workflows/.github/workflows/updater.yml@v1
+    with:
+      path: sentry-cli.properties
+      name: CLI
+    secrets:
+      # If a custom token is used instead, a CI would be triggered on a created PR.
+      # api_token: ${{ secrets.CI_DEPLOY_KEY }}
+      api_token: ${{ github.token }}
+
+  cocoa:
+    uses: getsentry/github-workflows/.github/workflows/updater.yml@v1
+    with:
+      path: scripts/update-cocoa.sh
+      name: Cocoa SDK
+    secrets:
+      # If a custom token is used instead, a CI would be triggered on a created PR.
+      # api_token: ${{ secrets.CI_DEPLOY_KEY }}
```

Comment:
That's strange. I've only seen `api_token: ${{ secrets.CI_DEPLOY_KEY }}` used in `update-deps.yml`.

Reply:
Both work but yeah `secrets.CI_DEPLOY_KEY` should be more flexible